pyserini/search/__main__.py

```diff
@@ -28,9 +28,17 @@
 parser.add_argument('--hits', type=int, metavar='num', required=False, default=1000, help="Number of hits.")
 parser.add_argument('--msmarco',  action='store_true', default=False, help="Output in MS MARCO format.")
 parser.add_argument('--output', type=str, metavar='path', help="Path to output file.")
+
+parser.add_argument('--max-passage',  action='store_true', default=False, help="Select only max passage from document.")
+parser.add_argument('--max-passage-hits', type=int, metavar='num', required=False, default=100,
```

Comment:
Is this argument being used? Same for `max-passage-delimiter`

Reply:
Yup, see comments below.